src/NServiceBus.AcceptanceTests/Encryption/EndpointTemplates/DefaultServer.cs

```diff
@@ -42,7 +42,8 @@ public BusConfiguration GetConfiguration(RunDescriptor runDescriptor, EndpointCo
             builder.CustomConfigurationSource(configSource);
             builder.EnableInstallers();
 
-            builder.DisableFeature<TimeoutManager>();
+            // TimeoutManager is currently required by Sagas
+            builder.EnableFeature<TimeoutManager>();
```

Comment:
Do we need to turn the timeoutmanager back on for all tests? (it does make tests quicker and less queues created)

Can't we just turn it on for the tests that needs the TM to be on?